app/cards/renaissance/villain.js

```diff
@@ -0,0 +1,60 @@
+Villain = class Villain extends Card {
+
+    types() {
+        return ['action', 'attack']
+    }
+
+    coin_cost() {
+        return 5
+    }
+
+    play(game, player_cards) {
+        if (game.turn.possessed) {
+            possessing_player_cards = PlayerCardsModel.findOne(game._id, game.turn.possessed._id)
+            possessing_player_cards.coin_tokens += 2
+            game.log.push(`&nbsp;&nbsp;<strong>${possessing_player_cards.username}</strong> takes a coin token`)
```

Comment:
I believe the log message needs to be updated to reflect the 2 coin tokens that are gained now instead of one